swagger/config.go

```diff
@@ -9,6 +9,8 @@ import (
 // PostBuildDeclarationMapFunc can be used to modify the api declaration map.
 type PostBuildDeclarationMapFunc func(apiDeclarationMap *ApiDeclarationList)
 
+type GetAdditionalSchemaFormat func(typeName string) string
```

Comment:
Suggestion to rename it as MapSchemaFormatFunc  ,  the other type also has the Func suffix and the acutal behavior is mapping (that could be implemented by a get on a hash)